crates/topos-p2p/src/constant.rs

```diff
@@ -1,5 +1,29 @@
+use std::env;
+
+use lazy_static::lazy_static;
+use prometheus_client::registry::Registry;
+use tokio::sync::Mutex;
+
 // TODO: Investigate BUFFER SIZE
-pub const EVENT_STREAM_BUFFER: usize = 2048;
-pub const COMMAND_STREAM_BUFFER: usize = 2048;
+
+lazy_static! {
+    pub static ref METRIC_REGISTRY: Mutex<Registry> = Mutex::new(<Registry>::with_prefix("topos"));
```

Comment:
doc comments would be great here to explain what the calculation does, why and what the constant is for?

Reply:
what do you mean by calculation?

Reply:
`.unwrap_or(2048 * 2);`

I think we should at least mention why this particular size etc.